uasurfer.go

```diff
@@ -122,6 +122,7 @@ const (
 	PlatformWindows
 	PlatformMac
 	PlatformLinux
+	PlatformAndroid
```

Comment:
don't put it in the middle, you've changed all IDS that's why I don't like iot for such consts